backends/arm/operators/op_avg_pool2d.py

```diff
@@ -52,7 +52,7 @@ def define_node(
         # Transpose to align with TOSA
         NHWC_Order = [0, 2, 3, 1]
         input_transposed = transpose_helper(
-            tosa_graph, input_tensor, NHWC_Order, output.dtype
+            tosa_graph, input_tensor.name, input_tensor.shape, NHWC_Order, output.dtype
```

Comment:
unrelated change?

Reply:
Nice catch. This change is unnecessary. Tidy it up in the following patch.